code/lib/cli/src/automigrate/fixes/missing-babelrc.ts

```diff
@@ -0,0 +1,91 @@
+import chalk from 'chalk';
+import dedent from 'ts-dedent';
+import semver from 'semver';
+import { getStorybookInfo } from '@storybook/core-common';
+import { loadPartialConfigAsync } from '@babel/core';
+import { readConfig } from '@storybook/csf-tools';
+import type { Fix } from '../types';
+
+interface MissingBabelRcOptions {
+  needsBabelRc: boolean;
+}
+
+const logger = console;
+
+const frameworksThatNeedBabelConfig = [
+  '@storybook/angular',
+  '@storybook/react-webpack5',
+  '@storybook/vue-webpack5',
+  '@storybook/vue3-webpack5',
+  '@storybook/preact-webpack5',
+  '@storybook/html-webpack5',
+  '@storybook/react-vite',
+  '@storybook/vue-vite',
+  '@storybook/vue3-vite',
+  '@storybook/preact-vite',
+  '@storybook/html-vite',
+];
+
+export const missingBabelRc: Fix<MissingBabelRcOptions> = {
+  id: 'missing-babelrc',
+  promptOnly: true,
+
+  async check({ packageManager }) {
+    const packageJson = packageManager.retrievePackageJson();
+    const { mainConfig, version: storybookVersion } = getStorybookInfo(packageJson);
+
+    const storybookCoerced = storybookVersion && semver.coerce(storybookVersion)?.version;
+    if (!storybookCoerced) {
+      throw new Error(dedent`
+        ❌ Unable to determine storybook version.
+        🤔 Are you running automigrate from your project directory?
+      `);
+    }
+
+    if (!semver.gte(storybookCoerced, '7.0.0')) {
+      return null;
+    }
+
+    if (!mainConfig) {
+      logger.warn('Unable to find storybook main.js config, skipping');
+      return null;
+    }
+
+    const main = await readConfig(mainConfig);
+
+    const frameworkField = main.getFieldValue(['framework']);
+    const frameworkPackage =
+      typeof frameworkField === 'string' ? frameworkField : frameworkField?.name;
+
+    if (frameworksThatNeedBabelConfig.includes(frameworkPackage)) {
+      const config = await loadPartialConfigAsync();
```

Comment:
This does not seem to be working, and I feel like I'm missing something quite simple..

Reply:
Fixed!